2020-09-04 02:40:22,675 No instance exit found for race: ELYOS 310050000
2020-09-04 02:42:54,514 Exception in a Runnable execution:
java.lang.NullPointerException: null
	at com.aionemu.gameserver.fix.GeoDataBugUtils.canSee(GeoDataBugUtils.java:17) ~[classes/:na]
	at com.aionemu.gameserver.skillengine.model.Effect.applyEffect(Effect.java:704) ~[classes/:na]
	at com.aionemu.gameserver.skillengine.model.Skill.applyEffect(Skill.java:716) ~[classes/:na]
	at com.aionemu.gameserver.skillengine.model.Skill$1.run(Skill.java:694) ~[classes/:na]
	at com.aionemu.commons.utils.concurrent.ExecuteWrapper.execute(ExecuteWrapper.java:30) ~[al-commons-1.3.jar:na]
	at com.aionemu.commons.utils.concurrent.RunnableWrapper.run(RunnableWrapper.java:22) [al-commons-1.3.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:476) [na:na]
	at java.util.concurrent.FutureTask.run(FutureTask.java:237) [na:na]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:151) [na:na]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:264) [na:na]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1115) [na:na]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:586) [na:na]
	at java.lang.Thread.run(Thread.java:745) [na:1.7.0_80]
